CDH-5760: Document that schema is defined by the first processed message (#4360)
SoftwareAG/c8y-docs: content/datahub/streaming-lake-ingestion-bundle/using.md

Currently, the service does not delete tables and columns when they are not in use anymore. For more information, see [Managing schema evolution](#managing-schema-evolution).
{{< /c8y-admon-info >}}

> {{ < c8y-admin-info>}}

Reviewer: ```suggestion
{{ < c8y-admon-info >}}
```
At least the previous block is using `c8y-admon-info` instead of `c8y-admin-info`.

The schema evolution guarantees automated conflict resolution of columns types and names.

> The base schema is defined by the first **processed** message; The following messages, if contain conflicting types or names

Reviewer 2: ```suggestion
The base schema is defined by the first **processed** message. The following messages, if they contain conflicting types or names,
```

will follow conflict resolution rules and be stored in overflow columns.

> However, there is no guarantee if messages are processed

Reviewer 2: ```suggestion
However, there is no guarantee that messages are processed
```

> exactly in order of arrival, we guarantee the order of processing for the same device.

Reviewer 2: ```suggestion
exactly in the order of arrival. We guarantee the order of processing for the same device.
```

In other words, if schema conflicts are introduced by messages from different devices, any device will  
define the resulting base schema.

> {{</ c8y-admin-info>}}

Reviewer: ```suggestion
{{</ c8y-admon-info >}}
```

The service currently does not support certain combinations of type conflicts with lists. It moves conflicting lists to the `trash` table, see [Binning](#binning).
{{< /c8y-admon-caution >}}

{{< c8y-admon-info >}}
The result of schema conflict resolution depends on the order in which messages are processed. If you first send a message with a numeric version property and then a message with a string version property, the result are two properties `version` (decimal) and `version__s`. If you reserve the order of the messages, the results are two properties `version` (string) and `version__d`. 

> Please note that the order of sending messages to Cumulocity and the order of processing messages does not necessarily correspond. For more details on ordering guarantees, please visit the [Messaging Service documentation](/api/core/#tag/About-notifications-2.0).

Reviewer 2: ```suggestion
Note that the order of sending messages to {{< product-c8y-iot >}} and the order of processing messages does not necessarily correspond. For more details on ordering guarantees, visit the [Messaging Service documentation](/api/core/#tag/About-notifications-2.0).
```
